ui/partials/main/thJobDetailsRetriggerMenu.html

```diff
@@ -1,6 +1,6 @@
 <li>
-  <a ng-click="retriggerJob([selectedJob])">Retrigger job</a>
+  <a ng-click="retriggerJob([selectedJob])" title="Re-run this job on this push">Retrigger job</a>
 </li>
 <li>
-  <a ng-click="backfillJob()">Backfill job</a>
+  <a ng-click="backfillJob()" title="Trigger this job on prior pushes to fill in the gap where jobs were coalesced.">Backfill job</a>
```

Comment:
How about:
`Trigger jobs of this type on prior pushes, to fill in gaps due to coalescing`
or
`Trigger jobs of this type on prior pushes, to fill in gaps where the job was not run`
or
`Trigger jobs of this type on prior pushes, to help with bisection`